_data/chains/eip155-8080.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Shardeum Liberty 1.3",
+  "name": "Shardeum Liberty 1.4",
```

Comment:
```suggestion
  "name": "Shardeum Liberty",
```

Reply:
Hi we have multiple test net running at a time with the milestones and different set of feature and version number are only the could differentiate.

Reply:
We currently have 1.4 and 2.0 running live right now under liberty namespace.